distributed/diagnostics/plugin.py

```diff
@@ -24,13 +24,11 @@
 class SchedulerPlugin:
     """Interface to extend the Scheduler
 
-    The scheduler operates by triggering and responding to events like
-    ``task_finished``, ``update_graph``, ``task_erred``, etc..
-
-    A plugin enables custom code to run at each of those same events.  The
-    scheduler will run the analogous methods on this class when each event is
-    triggered.  This runs user code within the scheduler thread that can
-    perform arbitrary operations in synchrony with the scheduler itself.
+    A plugin enables to run custom hooks when specific events occur.  The
```

Comment:
Minor wording comment

```suggestion
    A plugin enables custom hooks to run when specific events occur.  The
```